lib/advisory.js

```diff
@@ -242,11 +242,13 @@ class Advisory {
     // check the dependency of this version on the vulnerable dep
     // if we got a version that's not in the packument, fall back on
     // the spec provided, if possible.
-    const mani = this[_packument].versions && this[_packument].versions[version] ? this[_packument].versions[version] : {
-      dependencies: {
-        [this.dependency]: spec,
-      },
-    }
+    const mani = () => { this[_packument].versions && this[_packument].versions[version] ?
```

Comment:
hmm this might introduce linting issue, not sure about the rule used by npm, I would make it like this to comply with standard js rules:
```js
    const mani = this[_packument].versions && this[_packument].versions[version]
      ? this[_packument].versions[version]
      : {
          dependencies: {
            [this.dependency]: spec
          }
        }
```